### ISSUE TITLE
overflow of index.html

### ISSUE THREAD

**Issue by stla** [ramnathv/poirot#20]
Hello Ramnath,

After doing some tests on http://stla.github.io/Test_Poirot3/, my definitive Poirot blog will be http://stla.github.io/stlapblog/index.html.

At the bottom of the sidebar I have put some links, the list of links overflows and there's no scroll. How to solve that please ?

**Comment by ramnathv** [comment]
The sidebar is intended to be fixed. You will have to modify the CSS to make it scroll and that might introduce other issues. One option is to do the following.

Tweak [these lines](https://github.com/stla/stlapblog/blob/gh-pages/libraries/frameworks/purus/partials/sidebar.html#L16-L18) in your template by removing the `{{ site.url }}` part

``` html
{{# site.navigation }}
   <li><a href="{{site.url}}{{href}}">{{item}}</a></li>
{{/ site.navigation }}
```

and then in your [site.yml](https://github.com/stla/stlapblog/blob/gh-pages/site.yml#L10-L12) do

``` yaml
navigation:
  - {item: Home, href: "http://stla.github.io/stlapblog/index.html"}
  - {item: About, href: "http://stla.github.io/stlapblog/about.html"}
  - {item: Slidify, href: "http://slidify.org"}
```

This will render these links in the sidebar and will avoid the overflow issue.

Hope this helps.

**Comment by stla** [comment]
I'm afraid I have not totally understand your proposal. Am I supposed to remove `{{site.url}}` only ? When I'm on a post page, such as http://stla.github.io/stlapblog/posts/rgl3D.html, and when I click "Home", I am directed to the unvalid link `http://stla.github.io/stlapblog/posts/index.html`.

**Comment by stla** [comment]
Otherwise, maybe it would be nicer to have a right sidebar that belongs to the same block of each post page (there is a nice available space at right by the way).

**Comment by ramnathv** [comment]
Yes, the right sidebar sounds like a good idea. Let me see what is the easiest way to tweak the base layout to achieve this. I am on vacation from tomorrow, so this might take a while :)

**Comment by stla** [comment]
Ok, have fun !
